### PR TITLE
Create initial admin README

### PR DESCRIPTION
**Pull request by hyanwong**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+$ git push origin my-topic-branch
+# Go to github & open a PR based on my-topic-branch (the option appears automatically).
+```
+
+Python contributions should conform to [PEP-8](https://www.python.org/dev/peps/pep-0008/)
+with lines limited to a maximum of 89 characters, and no trailing whitespace.
+We prefer contributions to documentation to follow this style where possible.
+
+All submissions are accepted on the basis that they have been provided to the tskit
+project under the [MIT License](https://opensource.org/licenses/MIT).
```
Comment (jeromekelleher) [inline README.md]: I would change this to "the relevant OSS license" (as they differ).

File: README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+## Contributing
+
+We encourage contributions to improve any of the
+[tskit development repositories](https://github.com/tskit-dev).
+Improvements to documentation are particularly welcome, and are likely to be accepted
+quickly. 
+
+The preferred way to contribute is to create your own fork of the appropriate
+repository (for instance, `tskit-dev/tskit`), make changes, and submit your changes as a
+pull request (PR). A simple way to do this is to create a topic branch based on upstream,
```
Comment (jeromekelleher) [inline README.md]: We need to tell people how to set up an upstream remote. Perhaps point to docs here: https://help.github.com/articles/configuring-a-remote-for-a-fork/?

**Review by jeromekelleher — CHANGES_REQUESTED** [review]
Looks good, thanks @hyanwong. Some minor changes needed for precision.

File: README.md (hunk 1 above)
Comment (hyanwong) [reply README.md]: Ah, I didn't realise that it wasn't all under MIT. I think it's helpful for contributors to know roughly without hunting around, so perhaps "the OSS license in the relevant repository (usually MIT or XXX or XXX, as specified in the LICENSE file of the appropriate repository)"

File: README.md (hunk 2 above)
Comment (hyanwong) [reply README.md]: When I do this on GitHub (by "opening in desktop"), it seems to set the remote automatically. I'll phrase it appropriately

File: README.md (hunk 1 above)
Comment (jeromekelleher) [reply README.md]: I think there's GPL and maybe a Creative Commons. I don't really want to keep this up to date though, so don't want to give the impression the list is exhaustive. Maybe "usually MIT or GPL"?

Comment (hyanwong) [reply README.md]: Sounds fine to me. I agree about keeping it up to date. "Usually" should cover us here

**Review by jeromekelleher — APPROVED** [review]

**Comment by jeromekelleher** [comment]
Looks good, thanks.

**Pull request merged**